ensembl_dbcopy/migrations/0003_add_dry_run_fix_dbs_2_exclude.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.13 on 2020-11-19 16:44
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ensembl_dbcopy', '0002_adding_request_date'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='dbs2exclude',
+            name='id',
+        ),
+        migrations.AddField(
+            model_name='requestjob',
+            name='dry_run',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AlterField(
```

Comment:
Did you check that applying this migration don't delete the content of this table?

Reply:
Yes, I did.